### PR TITLE
Added Stork Longevity Stats and Refactored Code

### PR DESCRIPTION
**Pull request by gejain**
Added Stork Longevity Stats and Refactored Code

**What this PR does / why we need it**:
It will push longevity stats for Stork Triggers to Aetos

**Which issue(s) this PR fixes** (optional)
Closes #PTX-21932

**Special notes for your reviewer**:

Getting triggers stats like below in Aetos -

image

### CODE REVIEW

File: pkg/stats/stats.go [hunk 1]
```diff
@@ -66,6 +73,30 @@ func getRebootStats(rebootTime, nodeID, pxVersion string) (map[string]string, er
 	return rebootExportable, nil
 }
 
+func GetStorkMigrationStats(mig *storkv1.Migration) (sts map[string]string, err error) {
+	migStats := make(map[string]string)
+	migStats["MigrationName"] = mig.Name
+	migStats["MigrationNamespace"] = mig.Namespace
```
Comment (lsrinivas-pure) [inline pkg/stats/stats.go:79]: since all are constants, no need to return err here

Comment (kphalgun-px) [reply pkg/stats/stats.go:79]: There is no point in naming return values.

Comment (gejain) [reply pkg/stats/stats.go:79]: I changed the code at last moment and forgot to remove it, will update it.

Comment (gejain) [reply pkg/stats/stats.go:79]: Updated it, Thanks. Please review.

File: tests/testTriggers.go [hunk 2]
```diff
@@ -628,6 +628,11 @@ func endLongevityTest() {
 func updateLongevityStats(name, eventStatName string, dashStats map[string]string) {
 	name = strings.Split(name, "<br>")[0] //discarding the extra strings attached to name if any
 	version, err := Inst().V.GetDriverVersion()
+	product := "px-enterprise"
+	if eventStatName == stats.AsyncDREventName || name == stats.MetroDREventName || name == stats.StorkApplicationBackupEventName {
```
Comment (kphalgun-px) [inline tests/testTriggers.go:632]: It should be either eventStatName or name used consistently across the conditions, right?

File: tests/testTriggers.go [hunk 3]
```diff
@@ -8063,10 +8055,15 @@ func TriggerStorkAppBkpVolResize(contexts *[]*scheduler.Context, recordChan *cha
 					return
 				}
 				log.InfoD("backup successful and volume resize injected during backup successfully, backup name - %v, backup location - %v", backupname, backuplocationname)
+				dashStats, err := stats.GetStorkBackupStats(backupname, currbkNamespace)
+				if err != nil {
+					log.InfoD("Not able to get stats, err: %v", err)
+				}
+				updateLongevityStats(StorkAppBkpVolResize, stats.StorkApplicationBackupEventName, dashStats)
```
Comment (kphalgun-px) [inline tests/testTriggers.go:8062]: Is it stats.StorkApplicationBackupEventName? It does not look like it corresponds to TriggerStorkAppBkpVolResize.

**Review by kphalgun-px — APPROVED** [review]
LGTM

**Review by Rohit-PX — APPROVED** [review]
LGTM.

**Comment by Rohit-PX** [comment]
@gejain Please rebase with master branch before merging.

**Review by Rohit-PX — APPROVED** [review]

File: tests/testTriggers.go (hunk 2 above)
Comment (gejain) [reply tests/testTriggers.go:632]: Have grouped triggers in eventName so that we dont need to put all here. This should be ok.

**Pull request merged**